src/test/resources/code-samples/valid/ValidFile.kt

```diff
@@ -0,0 +1 @@
+package `code-samples`.`valid`
```

Comment:
Created a valid input to execute a full testrun. Maybe this is not necessary - not sure.